apps/admin/frontend/src/components/confirm_removing_file_modal.tsx

```diff
@@ -70,7 +70,7 @@ export function ConfirmRemovingFileModal({
 
   return (
     <Modal
-      centerContent
```

Comment:
Seems like we more often left-align our modals than center-align them, so I've made that consistent for CVR modals.